tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature

```diff
@@ -0,0 +1,66 @@
+Feature: copying file using file id
+  As a user
+  I want to copy the file using file id
+  So that I can manage my resource
+
+  Background:
+    Given using spaces DAV path
+    And user "Alice" has been created with default attributes and without skeleton files
+
+
+  Scenario Outline: copy a file from folder to the root of personal space
+    Given user "Alice" has created folder "/folder"
+    And user "Alice" has uploaded file with content "some data" to "/textfile.txt"
+    And we save it into "FILEID"
+    When user "Alice" copies a file with FILEID in URL "<dav-path>" to "/folder/textfile.txt" inside space "Personal"
+    Then the HTTP status code should be "201"
+    And for user "Alice" the space "Personal" should contain these entries:
+      | folder/textfile.txt |
```

Comment:
also check the original file exists since it is a COPY request
```feature
And for user "Alice" the space "Personal" should contain these entries:
      | textfile.txt |
```